common-travel.lic

```diff
@@ -75,10 +75,7 @@ module DRCT
   def retreat(ignored_npcs = [])
     return if (DRRoom.npcs - ignored_npcs).empty?
```

Comment:
❓ Why wasn't this moved?

Reply:
do we want to add drinfomon to custom_require in common.lic?

Reply:
Probably the best choice.

Reply:
I'm an idiot, was thinking adding drinfomon to common would add another circular dependency but drinfomon doesn't include common, and common already includes drinfomon. fixed.